docs/07_Installing_Essential_Cluster_Services.md

```diff
@@ -117,6 +117,8 @@ In the toplevel project directory (`kubenet`) on the host machine, create a dire
 
 ```bash
 mkdir nfs-pvs
+sudo chown nobody:nogroup nfs-pvs    # user:group does not matter
+sudo chmod 2770 /data/nfs            # sticky-bit on
```

Comment:
What's exactly the purpose of this? Just to simplify NFS setup?

Reply:
yes, the provisioner could not create the volumes. Line 121 should have been `sudo chmod 2770 nfs-pvs`